tokio/src/process/mod.rs

```diff
@@ -839,15 +839,35 @@ pub struct Child {
     child: FusedChild,
 
     /// The handle for writing to the child's standard input (stdin), if it has
-    /// been captured.
+    /// been captured. To avoid partially moving the `child` and thus blocking
+    /// yourself from calling functions on `child` while using `stdin`, you might
+    /// find it helpful to do:
+    ///
+    /// ```compile_fail,E0425
+    /// let stdin = child.stdin.take().unwrap();
+    /// ```
```

Comment:
Let's not have tests that fail to compile.
```suggestion
    /// ```no_run
    /// # let child: tokio::process::Child = unreachable!();
    /// let stdin = child.stdin.take().unwrap();
    /// ```
```

Reply:
Using the code suggested errors because the second line in unreachable, what do prefer to do? Set it to `ignore` rather than `no_run`?

Reply:
Just throw in some dummy command instead.
```Rust
let child = tokio::process::Command::new("echo").spawn().unwrap();
```

Reply:
https://github.com/tokio-rs/tokio/pull/3437/commits/3670306cd212666a78c22230555d5d803e70b8ce